plugins/LanguagesManager/tests/UI/LanguagesManager_TopNavigation_spec.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Language selector top navigation screenshot tests for anonymous users.
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe('LanguagesManager_TopNavigation', function () {
+    this.fixture = "Piwik\\Tests\\Fixtures\\UITestFixture";
```

Comment:
To avoid a possible manipulation of the fixture and to speed things up I would suggest to use a small fixture and disable the data persistence.
```suggestion
    this.fixture = "Piwik\\Tests\\Fixtures\\OneVisit";
    this.optionsOverride = {
        'persist-fixture-data': false
    };
```

Then you can also remove the API call afterwards to unset the anonymous access.